internal/capability/operand_cleanup.go

```diff
@@ -0,0 +1,67 @@
+package capability
+
+import (
+	"context"
+	"fmt"
+	"log"
+
+	"github.com/opdev/opcap/internal/operator"
+
+	apiextensionsv1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"
+
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+)
+
+// OperandCleanup removes the operand from the OCP cluster in the ca.namespace
+func (ca *capAudit) OperandCleanUp() error {
+	logger.Debugw("cleaningUp operand for operator", "package", ca.subscription.Package, "channel", ca.subscription.Channel, "installmode", ca.subscription.InstallModeType)
```

Comment:
Just a question: how do we activate the debug mode for logging? Is this writing out somewhere?

Reply:
In the `internal/logger/zap.go` file, we can change the level to `"level":"debug",` to allow zap to store debug logs. https://github.com/opdev/opcap/blob/0e1db4aa57b9714e0a2a0d004ce2583cce933806/internal/logger/zap.go#L17

Reply:
Great! I just opened #132 for this. Option for having it on a flag.